ccmlib/node.py

```diff
@@ -1254,22 +1257,34 @@ def _update_pid(self, process):
             raise NodeError('Problem starting node %s' % self.name, process)
         self.__update_status()
 
-    def __gather_sstables(self, datafile=None, keyspace=None, columnfamilies=None):
-        datafiles = []
+    def __gather_sstables(self, datafiles=None, keyspace=None, columnfamilies=None):
+        files = []
         if keyspace is None:
-            for k in self.list_keyspaces():
-                datafiles = datafiles + self.get_sstables(k, "")
-        elif datafile is None:
+            if datafiles:
+                for datafile in datafiles:
+                    if not os.path.isabs(datafile):
+                        datafile = os.path.join(os.getcwd(), datafile)
+
+                    if not os.path.exists(datafile):
+                        print_("File doesn't exist: " + datafile)
+                    elif datafile.endswith('Data.db') or datafile.endswith('big-Data.db'):
```

Comment:
The main reason behind this is to create the same behavior than if we called sstablesplit only one time. With a single sstablesplit call, we can pass something like *9998* as parameter and this will result in only 1 sstable split. If we try that with ccm, this will result in 6 sstable splits.